### PR TITLE
[ci] openssh: security update to 8.8p1

### LINKED ISSUE #3546: openssh: CVE-2021-41617
### CVE IDs

CVE-2021-41617

### Other security advisory IDs

[OpenSSH Security](https://www.openssh.com/security.html)

### Description

sshd in OpenSSH between 6.2 and 8.7 (inclusive).
sshd(8) failed to correctly initialise supplemental groups when executing an AuthorizedKeysCommand or AuthorizedPrincipalsCommand, where a AuthorizedKeysCommandUser or AuthorizedPrincipalsCommandUser directive was been set to run the command as a non-root user. Instead these commands would inherit the groups that sshd(8) was started with.
Depending on system configuration, inherited groups may allow the helper programs to gain unintended privilege.
Neither AuthorizedKeysCommand nor AuthorizedPrincipalsCommand are enabled by default in sshd_config(5).
This bug is corrected in OpenSSH 8.8
For more information, please refer to the [release notes](https://www.openssh.com/txt/release-8.8).

### Patches

N/A

### PoC(s)

N/A

### PR DESCRIPTION
**Pull request by KexyBiscuit**
Topic Description
-----------------

- Patch `ssh_config` to support drop-ins. This enables packaging of Buildbots.
- Lint scripts.

Package(s) Affected
-------------------

`openssh`

Security Update?
----------------

Yes

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`
    

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [ ] PowerPC 64-bit (Little Endian) `ppc64el`

Update(s) Uploaded to Stable
----------------------------

**Primary Architectures**

- [x] AMD64 `amd64`   
- [x] AArch64 `arm64`

**Secondary Architectures**

Architectural progress for "secondary," or experimental ports does not impede on merging of this topic.

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`

### CODE REVIEW

File: extra-network/openssh/autobuild/beyond [hunk 1]
```diff
@@ -1,7 +1,15 @@
-install -v -m755 contrib/ssh-copy-id "$PKGDIR"/usr/bin
-install -Dm755 contrib/findssl.sh "$PKGDIR"/usr/bin/findssl.sh
-install -Dm644 contrib/ssh-copy-id.1 "$PKGDIR"/usr/share/man/man1/ssh-copy-id.1
+abinfo "Installing contrib files ..."
+install -Dvm755 contrib/ssh-copy-id "$PKGDIR"/usr/bin/ssh-copy-id
```
Comment (MingcongBai) [inline extra-network/openssh/autobuild/beyond]: Use absolute path.

File: extra-network/openssh/autobuild/beyond [hunk 2]
```diff
@@ -1,7 +1,15 @@
...
+install -Dvm755 contrib/findssl.sh "$PKGDIR"/usr/bin/findssl.sh
+install -Dvm644 contrib/ssh-copy-id.1 "$PKGDIR"/usr/share/man/man1/ssh-copy-id.1
 
+abinfo "Patching ssh_config ..."
```
Comment (MingcongBai) [inline extra-network/openssh/autobuild/beyond]: Be specific about what you did here.

File: extra-network/openssh/autobuild/beyond [hunk 3]
```diff
@@ -1,7 +1,15 @@
...
+sed -i 's/# Host \*/Host \*/' "$PKGDIR"/etc/ssh/ssh_config
+cat << EOF >> "$PKGDIR"/etc/ssh/ssh_config
+  Include ssh_config.d/*
+EOF
+
+abinfo "Patching sshd_config ..."
```
Comment (MingcongBai) [inline extra-network/openssh/autobuild/beyond]: Ditto above.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

**Review by MingcongBai — APPROVED** [review]

**Comment by KexyBiscuit** [comment]
Smoke test passed.

**Pull request merged**